<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ONF_ActionDefineHeightLayerOptions</class>
 <widget class="QWidget" name="ONF_ActionDefineHeightLayerOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wd_toolbar" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pb_min">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_max">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_plus">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_minus">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/moins.png</normaloff>:/icons/moins.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_m">
           <property name="text">
            <string>1 m</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_singleStep</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_dm">
           <property name="text">
            <string>10 cm</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_singleStep</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_cm">
           <property name="text">
            <string>1 cm</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_singleStep</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_mm">
           <property name="text">
            <string>1 mm</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_singleStep</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_sideView">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/side_view.png</normaloff>:/icons/side_view.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rb_zmin">
           <property name="text">
            <string>Zmin</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_z</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_zmin">
           <property name="suffix">
            <string> m</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_zmax">
           <property name="text">
            <string>Zmax</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_z</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_zmax">
           <property name="suffix">
            <string> m</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_computeCrownProjection">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Seuil</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_threshold">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>RÃ©solution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_resolution">
           <property name="suffix">
            <string> m</string>
           </property>
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_z"/>
  <buttongroup name="bg_singleStep"/>
 </buttongroups>
</ui>
